### PR TITLE
Fixed indentation and comments

### PR DESCRIPTION
**Pull request by SwathiPottigari**
@CodingDawg Please review.

### CODE REVIEW

File: MergeSortedArrays.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ public class Solution {
                  n--;
              }
             }
-		}
+	}
```
Comment (codingdawg) [inline MergeSortedArrays.cs]: The indentation is still not good. Please fix it.

File: MergeSortedArrays.cs [hunk 2]
```diff
@@ -1,5 +1,5 @@
 public class Solution {
-	//SOlution to LeetCode problem.
+	//Solution to LeetCode problem.
     public void MergeSortedArrays(int[] nums1, int m, int[] nums2, int n) {
```
Comment (codingdawg) [inline MergeSortedArrays.cs]: Can you add some comments and describe what this method is doing.

File: MergeSortedArrays.cs [hunk 3]
```diff
@@ -1,5 +1,5 @@
 public class Solution {
-	//SOlution to LeetCode problem.
```
Comment (codingdawg) [inline MergeSortedArrays.cs]: Can you also put a link to the leet code problem here.

Comment (codingdawg) [reply MergeSortedArrays.cs]: Can you also add Test cases and results to this problem.

**Comment by SwathiPottigari** [comment]
@codingdawg  Please review now

**Comment by SwathiPottigari** [comment]
@codingdawg  Please review now

File: MergeSortedArrays.cs [hunk 4]
```diff
@@ -1,5 +1,7 @@
 public class Solution {
 	//SOlution to LeetCode problem.
+	//https://leetcode.com/problems/merge-sorted-array/description/
+	/* This method merges two sorted arraya*/
```
Comment (codingdawg) [inline MergeSortedArrays.cs]: arraya  typed incorrectly.

Comment (SwathiPottigari) [reply MergeSortedArrays.cs]: corrected the comment

**Pull request merged**